spec/factories/claims.rb

```diff
@@ -34,6 +34,11 @@
       association(:check, factory: [:check, :approved], strategy: :build)
     end
 
+    trait :payrolled do
```

Comment:
I'm not sure how I feel about this trait. It's presence invites use, but I'm not sure we'd really want to encourage creating claims with this trait as each one will be associated with its own payroll run, which a) wouldn't be very efficient, and b) wouldn't reflect real-world usage. Maybe it's better to not define this trait and just explicitly set the `payroll_run` inline in the one spec where it is being used?